backend/app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -77,7 +77,7 @@ def load_taxon
       end
 
       def load_taxonomy
-        @taxonomy = Taxonomy.find(params[:taxonomy_id])
```

Comment:
@imgarylai and also here you need to add that check for `defined?(SpreeGlobalize)` - otherwise, this will fail, because not all Spree installations are using that gem

Reply:
@damianlegawiec Thank you to point it out! Let me fix them.